### PR TITLE
Bump + pin web versions

### PR DESCRIPTION
**Pull request by illusional**
Pin with `pip-compile` and install with no-deps to completely pin web versions.
(Sorry, I included a small doc change)

### CODE REVIEW

File: dataproc/README.md [hunk 1]
```diff
@@ -66,9 +66,11 @@ Note, we support specifying the hail_version, but only a very select number of v
 ### Rebuilding the dataproc driver image
 
 ```sh
+# from repo root
+cd dataproc
 gcloud config set project analysis-runner
 # grab the HAIL_VERSION from analysis_runner/dataproc.py
-HAIL_VERSION=$(grep "DEFAULT_HAIL_VERSION = '" analysis_runner/dataproc.py | awk -F\' '{print $2}')
+HAIL_VERSION=$(grep "DEFAULT_HAIL_VERSION = '" ../analysis_runner/dataproc.py | awk -F\' '{print $2}')
```
Comment (MattWellie) [inline dataproc/README.md:73]: could shuffle this line to before `cd dataproc` and skip the `../`

File: web/main.py [hunk 2]
```diff
@@ -5,12 +5,12 @@
...
 import os
-from flask import Flask, abort, request, Response, stream_with_context
 
-from cpg_utils.cloud import read_secret, is_member_in_cached_group
-import google.cloud.storage
 import google.auth.transport.requests
+import google.cloud.storage
 import google.oauth2.id_token
+from cpg_utils.cloud import is_member_in_cached_group, read_secret
+from flask import Flask, Response, abort, request, stream_with_context
```
Comment (MattWellie) [inline web/main.py:13]: total nit, but is this post `isort`? flask should be before internal libs, but it won't be recognising cpg-utils as internal

**Review by MattWellie — APPROVED** [review]
nits, wasn't nearly as scary as the line count suggested

File: web/main.py (hunk 2 above)
Comment (illusional) [reply web/main.py:13]: Got this incoming soon: https://github.com/populationgenomics/cpg-python-template-repo/pull/12#issuecomment-1881917977
That should sort our isort funs

File: dataproc/README.md (hunk 1 above)
Comment (illusional) [reply dataproc/README.md:71]: Aye aye, good shout

**Pull request merged**